proxy/src/logging.rs

```diff
@@ -40,31 +48,30 @@ pub fn init() {
         .init();
 }
 
-/// Execute a closure with a `Debug` item attached to allow log messages.
+/// Execute a closure with a `Display` item attached to allow log messages.
```

Comment:
Comment is out of date (and those below as well).

Reply:
Sorry, it's no longer wrong. I was just noting it said Display when the PR originally required the LogCtx trait.